TestPlan.md

```diff
@@ -162,7 +178,8 @@ The tree view of maven projects should update when any pom.xml is created/modifi
     "maven.terminal.useJavaHome": true      // Use the Red Hat Java Language Support Setting for JAVA_HOME
     ```
 4. Trigger a maven command, in the newly created terminal, verify:
-    1. value of env `JAVA_HOME` should be the value you just set in `java.home`
+    1. On windows platform, input `$Env:MAVEN_OPTS` and press `Enter`, the value of env `JAVA_HOME` should be the value you just set in `java.home`.

```

Comment:
`$Env:MAVEN_OPTS` only works for PowerShell. I remember it doesn't apply for CMD. 

windows => Windows